### PR TITLE
 Added changes to labels.json  file

### PR DESCRIPTION
**Pull request by arinkulshi-skylight**
# Pull Request

## Description
Added labels.json edits 
https://github.com/CDCgov/IDWA/issues/106

- [x] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [x] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [x] I have updated the documentation, if applicable.
- [x] I have added or updated test cases to cover my changes, if applicable.
- [x] I have minimized the number of reviewers to include only those essential for the review.
- [x] I have notified teammates in the review thread to build awareness.

## Checklist for Reviewers
Please review and complete the following checklist during the review process:

- [ ] The code follows best practices and conventions.
- [ ] The changes implement the desired functionality or fix the reported issue.
- [ ] The tests cover the new changes and pass successfully.
- [ ] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

File: OCR/ocr/services/pdf_field_extractor.py [hunk 1]
```diff
@@ -43,10 +43,10 @@ def close_reader(self) -> None:
 
     def list_annotations(self):
         """
-         Generates a list of annotations in the pdf provided this method is mostly used for debugging purposes
+        Generates a list of annotations in the PDF. This method is mostly used for debugging purposes.
 
         Returns:
-            str: A string list of the type of annotations the name of the fields and its coordinates on the page
+            str: A string list of the type of annotations, the name of the fields, its coordinates on the page, and the specific widget type (checkbox or text box).
```
[the remark this answers is not in the document]
Comment (arinkulshi-skylight) [reply OCR/ocr/services/pdf_field_extractor.py]: I have edited the comment here. Let me know if this works.

File: OCR/ocr/services/pdf_field_extractor.py [hunk 2]
```diff
@@ -59,8 +59,13 @@ def list_annotations(self):
...
                 field_name = annot.get("/T")
                 rect = annot.get("/Rect")
-                subtype = annot.get("/Subtype")
-                print(f"Annotation - Type: {subtype}, Field Name: {field_name}, Coordinates: {rect}")
+                field_type = annot.get("/FT")
+                widget_type = "Unknown"
+                if field_type == pypdf.generic.NameObject("/Btn"):
+                    widget_type = "Checkbox"
+                elif field_type == pypdf.generic.NameObject("/Tx"):
+                    widget_type = "Text Box"
```
[the remark this answers is not in the document]
Comment (arinkulshi-skylight) [reply OCR/ocr/services/pdf_field_extractor.py]: Good idea I have made this change.
